src/Components/Section/Section.js

```diff
@@ -0,0 +1,22 @@
+import React, {Fragment} from 'react';
+import { HotelCard } from '../HotelCard';
+import {hotels} from '../HotelCard/config'
+
+import './Section.css'
+
+export const Section = ({h2Text}) => (
+  <section className="homes">
+
+    <h2 className="h2-text">{h2Text}</h2>
+    <div className="homes__examples">
+
+      { hotels.map((item) => (
+        <Fragment key={item.id}>
+          <HotelCard  {...item} />
+        </Fragment>
```

Comment:
There is no need for React.Fragment to return one element only.

Reply:
Done